pkg/gateway/model_build_targetgroup_test.go

```diff
@@ -316,9 +314,8 @@ func Test_TGModelByHTTPRouteBuild(t *testing.T) {
 			name: "Create LatticeService where backend mcs serviceimport does NOT exist",
 			httpRoute: &gateway_api.HTTPRoute{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:              "service4",
-					Finalizers:        []string{"gateway.k8s.aws/resources"},
-					DeletionTimestamp: &now,
+					Name:       "service4",
+					Finalizers: []string{"gateway.k8s.aws/resources"},
 				},
```

Comment:
When test httproute creation (wantIsDeleted is false),  HTTPRoute's DeletionTimestamp should be 0